src/components/hint.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Hint extends Component {
+  render() {
+    const hint = this.props.hint;
+    if (hint.get('status') === "yes please") {
+      let people = this.props.person.slice().sort(function(a,b) {
+        return Math.random() - Math.random();
```

Comment:
In our discussion, we were having a hard time figuring out if this does what we want. Is there a more conceptually simple way to express this?